test/browser/amd/index.html

```diff
@@ -1,43 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Now gulp test can testing things properly, we have need to support the browser test anymore.
If testing in browser is necessary, we could use krama instead